packages/graphql/src/services/practiceQuizzes.ts

```diff
@@ -247,6 +251,126 @@ function updateSpacedRepetition({
   }
 }
 
+function computeNewAverageTimes({
+  existingInstance,
+  existingResponse,
+  answerTime,
+}: {
+  existingInstance: ElementInstance & {
+    instanceStatistics: InstanceStatistics | null
```

Comment:
probably would be better to say the param can't be null and solving the type issue outside than | null and then ! on it?